### PR TITLE
Show error when we fail to install a GitHub webhook

### LINKED ISSUE #371: Add  GItHub project fails with 500 if we cannot install install webhooks
Title says all.

We should handle it gracefully and report an error to the user

### PR DESCRIPTION
**Pull request by agathver**
Closes #371

### CODE REVIEW

File: site/app/Resources/views/layout.html.twig [hunk 1]
```diff
@@ -22,8 +22,11 @@
   {% if app.request.hasPreviousSession %}
     {% for type, messages in app.session.flashbag.all() %}
       {% for message in messages %}
-        <div class="alert alert-{{ type }}" role="alert">
-          {{ message|raw }}
+        <div class="alert alert-{{ type }} alert-dismissable" role="alert">
+          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+          {{ message | trans | raw }}
```
Comment (imphil) [inline site/app/Resources/views/layout.html.twig]: We don't have translated strings anywhere, just hardcode it here in English.

File: site/src/Controller/ProjectController.php [hunk 2]
```diff
@@ -162,13 +164,17 @@ public function newAction(
...
-            );
+            try {
+                $this->populateProjectFromForm(
+                    $p,
+                    $form,
+                    $githubApiService,
+                    $organizationRepository,
+                    $userManager
+                );
+            } catch (ErrorException $e) {
```
Comment (imphil) [inline site/src/Controller/ProjectController.php]: Can we find a more focused Exception to catch, or event give the user bits of the error message we got from GitHub?

File: site/app/Resources/views/layout.html.twig (hunk 1 above)
Comment (agathver) [reply site/app/Resources/views/layout.html.twig]: I use some translations as the lines were being too long when included in code and were a bit testing-unfriendly

**Comment by imphil** [comment]
Thanks, LGTM!

**Pull request merged**